README.md

```diff
@@ -205,6 +205,32 @@ Once traces have been generated, you can view them using the
 Copy `OpenCL.dll` into a system location or alongside the application executable
 you want to run with clvk.
 
+### Raspberry Pi
+
+Make sure you have an up-to-date Mesa installed on your system.
+At the time of writing (May 2023) RaspberryPi OS (Debian 11) did not, but Ubuntu
+23.04 does have a compatible vulkan driver.
+
+Install the prerequisites:
+```
+$ sudo apt install mesa-vulkan-drivers vulkan-tools libvulkan-dev git cmake clang clinfo
+```
+
+Check if your vulkan implementation has the `VK_KHR_storage_buffer_storage_class`
+extension. Note that it's not enough if this extension shows up for llvmpipe.
+You need v3dv to have this extension.  If it is not there, your Mesa is too old.
+
+Before you run `./external/clspv/utils/fetch_sources.py` you need to edit its
+first line. Change `python` into `python3` so that the interpreter will be found.
+
+Building will take a very long time on a rPi4.
+Maybe you can skip some tool building, but building with default settings works, at least.
```

Comment:
I don't know how much time you can get, but here is what [ChromeOS](https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+/main/media-libs/clvk/clvk-0.0.1.ebuild) is using to reduce the build time of clvk:
```
-DLLVM_INCLUDE_BENCHMARKS=OFF
-DLLVM_INCLUDE_EXAMPLES=OFF
-DLLVM_INCLUDE_TESTS=OFF
-DLLVM_ENABLE_BINDINGS=OFF
-DLLVM_ENABLE_UNWIND_TABLES=OFF
-DLLVM_BUILD_TOOLS=OFF
-DCLSPV_BUILD_SPIRV_DIS=OFF
-DCLSPV_BUILD_TESTS=OFF
-DCLVK_BUILD_TESTS=OFF
-DCLVK_BUILD_SPIRV_TOOLS=OFF
-DCLVK_ENABLE_SPIRV_IL=OFF
```
 Maybe we should regroup all that under one variable inside our cmake. It would make it easier to maintain

Reply:
Some of these could probably be set by default. We could introduce a flag to select "dev" vs. "release" builds (that would have to be different from CMAKE_BUILD_TYPE I think) to enable/disable testing and tools maybe. AFAICT, the biggest savings will come from disabling SPIRV-Tools and the SPIRV-LLVM-Translator. Let's take the discussion to a dedicated issue to not pollute that PR.